modules/sde-common/src/main/java/org/eclipse/tractusx/sde/common/entities/SubmodelJsonReq.java

```diff
@@ -0,0 +1,48 @@
+/********************************************************************************
+ * Copyright (c) 2022, 2023 T-Systems International GmbH
+ * Copyright (c) 2022, 2023 Contributors to the Eclipse Foundation
```

Comment:
please check all files for 2024 copyright

Reply:
Updated